pkgs/development/tools/rust/cargo-watch/default.nix

```diff
@@ -1,26 +1,31 @@
-{ stdenv, lib, rustPlatform, fetchFromGitHub, CoreServices, rust }:
+{ stdenv, lib, rustPlatform, fetchFromGitHub, CoreServices, libiconv, rust }:
 
 rustPlatform.buildRustPackage rec {
   pname = "cargo-watch";
-  version = "7.6.1";
+  version = "7.7.0";
 
   src = fetchFromGitHub {
     owner = "passcod";
     repo = pname;
     rev = "v${version}";
-    sha256 = "sha256-vjX8xfwv/DOogji+OQCB9l5ebGBNoLW722TGpZ5Wg80=";
+    sha256 = "sha256:11sz55pf5hnmd0iq8wcdkn1dhrrpgnwzv93z3bj2di09v7mld0fg";
   };
 
-  cargoSha256 = "sha256-ku+tI0DIofV0EZ413sPjbJDUSqwTxiT8NWBeURrJW1k=";
+  cargoSha256 = "sha256:0p3wqald5ma3iw2nkk5wm62g6krmyzii65y3swjwi7j98rh1jgwq";
 
-  buildInputs = lib.optional stdenv.isDarwin CoreServices;
+  buildInputs = lib.optional stdenv.isDarwin [
```

Comment:
Does this not need to be `lib.optionals`?

```nix
  buildInputs = lib.optionals stdenv.isDarwin [ CoreServices libiconv ];
```

Reply:
Whelp, looks like it works!

Reply:
I'll admit I don't know the difference, but `lib.optional` seems to work in both cases.
I did find this: https://discourse.nixos.org/t/lib-optional-considered-harmful/7081 which seems to point to `lib.optional` 

Poking around some more, looks like `lib.optional` implicitly wraps its final argument up as a list, which would result in `lib.optionals stdenv.isDarwin [ CoreServices libiconv ]` evaluating to `[ [ CoreServices libiconv ] ]`. I'll replace `lib.optional` with `lib.optionals`.